nginx/main.conf

```diff
@@ -0,0 +1,30 @@
+env API_HOST;
+env ACCESS_TOKEN;
+env APICAST_PRODUCTION_ENDPOINT;
+env APICAST_STAGING_ENDPOINT;
+env APICAST_PORT;
+
+master_process 'on';
+worker_processes 'auto';
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  lua_code_cache 'on';
```

Comment:
Not really sure if nginx swallows the quotes. It is usual not not have them around.